### PR TITLE
Release 029

### PR DESCRIPTION
**Pull request by SpheMakh**
- Release 0.2.9 is ready
- A few more tests, and hopefully release on Friday (9/06/17)

### CODE REVIEW

**Comment by SpheMakh** [comment]
Don't merge. Failing containers are still not being caught properly

**Comment by SpheMakh** [comment]
retest this please

**Comment by SpheMakh** [comment]
retest this please

**Comment by bennahugo** [comment]
retest this please

**Comment by bennahugo** [comment]
@SpheMakh Can you attach the new test script please

File: stimela/cargo/cab/casa_setjy/parameters.json [hunk 1]
```diff
@@ -139,36 +139,22 @@
...
             "default": false, 
             "name": "listmodels"
         }, 
         {
-            "info": "Will create if necessary and use the MODEL_DATA", 
-            "dtype": "bool", 
-            "default": false, 
-            "name": "usescratch"
+            "info": "File location for field model", 
+            "dtype": "str",
```
Comment (bennahugo) [inline stimela/cargo/cab/casa_setjy/parameters.json]: Are you sure this isn't supposed to be an IO type?

**Comment by bennahugo** [comment]
@SpheMakh okay updated the test to include lwimager and ddfacet. Also added the beam into the mix on the second selfcal. Lets see if she runs through

**Comment by bennahugo** [comment]
retest this please

**Comment by bennahugo** [comment]
@SpheMakh Okay it does not register the output directory correctly. Currently the workspace only contains "output" not " output_ngc147_reduction". I've tried moving setting output to before registering globals but that doesn't work either.

**Comment by bennahugo** [comment]
@SpheMakh Okay attached the Testing script into the misc directory so it can be modified outside of jenkins. Makes life a bit easier.

**Pull request merged**